From: Martin Loschwitz <madkiss@debian.org>
Date: Thu, 02 Oct 2014 15:27:00 +0000
Subject: Turn var/drbd.d into var/lib/drbd.d

Last-Update: 2014-10-07
Forwarded: no
---

diff -ruN drbdmanage-0.20~pre1+git.f925f91.orig/conf/drbdmanaged.conf drbdmanage-0.20~pre1+git.f925f91/conf/drbdmanaged.conf
--- drbdmanage-0.20~pre1+git.f925f91.orig/conf/drbdmanaged.conf	2014-10-07 07:34:38.599142686 +0000
+++ drbdmanage-0.20~pre1+git.f925f91/conf/drbdmanaged.conf	2014-10-07 07:35:31.617823545 +0000
@@ -98,7 +98,7 @@
 #
 # default: /var/drbd.d
 # ============================================================
-drbd-conf-path  = /var/drbd.d
+drbd-conf-path  = /var/lib/drbd.d
 
 # ============================================================
 # VOLUME GROUP FOR THE DRBDMANAGE CONTROL VOLUME
@@ -108,4 +108,4 @@
 #
 # default: drbdpool
 # ============================================================
-drbdctrl-vg     = drbdpool
\ No newline at end of file
+drbdctrl-vg     = drbdpool
diff -ruN drbdmanage-0.20~pre1+git.f925f91.orig/conf/drbdmanage-resources.res drbdmanage-0.20~pre1+git.f925f91/conf/drbdmanage-resources.res
--- drbdmanage-0.20~pre1+git.f925f91.orig/conf/drbdmanage-resources.res	2014-10-07 07:34:38.599142686 +0000
+++ drbdmanage-0.20~pre1+git.f925f91/conf/drbdmanage-resources.res	2014-10-07 07:45:03.786792121 +0000
@@ -2,4 +2,4 @@
 # drbdmanage temporary resource configuration files
 # ============================================================
 
-include "/var/drbd.d/*.res";
\ No newline at end of file
+include "/var/lib/drbd.d/*.res";
diff -ruN drbdmanage-0.20~pre1+git.f925f91.orig/drbdmanage/server.py drbdmanage-0.20~pre1+git.f925f91/drbdmanage/server.py
--- drbdmanage-0.20~pre1+git.f925f91.orig/drbdmanage/server.py	2014-10-07 07:34:38.599142686 +0000
+++ drbdmanage-0.20~pre1+git.f925f91/drbdmanage/server.py	2014-10-07 07:36:10.163772667 +0000
@@ -108,7 +108,7 @@
         KEY_MAX_PORT_NR    : str(DEFAULT_MAX_PORT_NR),
         KEY_DRBDADM_PATH   : "/usr/sbin",
         KEY_EXTEND_PATH    : "/sbin:/usr/sbin:/bin:/usr/bin",
-        KEY_DRBD_CONFPATH  : "/var/drbd.d",
+        KEY_DRBD_CONFPATH  : "/var/lib/drbd.d",
         KEY_DRBDCTRL_VG    : DEFAULT_VG
     }
 
diff -ruN drbdmanage-0.20~pre1+git.f925f91.orig/man-pages/drbdmanage.xml drbdmanage-0.20~pre1+git.f925f91/man-pages/drbdmanage.xml
--- drbdmanage-0.20~pre1+git.f925f91.orig/man-pages/drbdmanage.xml	2014-10-07 07:34:38.599142686 +0000
+++ drbdmanage-0.20~pre1+git.f925f91/man-pages/drbdmanage.xml	2014-10-07 07:36:28.512700523 +0000
@@ -1093,7 +1093,7 @@
                 the local node are exported.
                 The configuration files will be created (or updated) in
                 the drbdmanage directory for temporary configuration files,
-                typically <option>/var/drbd.d</option>.
+                typically <option>/var/lib/drbd.d</option>.
             </para>
           </listitem>
         </varlistentry>
diff -ruN drbdmanage-0.20~pre1+git.f925f91.orig/setup.py drbdmanage-0.20~pre1+git.f925f91/setup.py
--- drbdmanage-0.20~pre1+git.f925f91.orig/setup.py	2014-10-07 07:34:38.599142686 +0000
+++ drbdmanage-0.20~pre1+git.f925f91/setup.py	2014-10-07 07:37:03.590474351 +0000
@@ -84,7 +84,7 @@
                 ("/usr/share/man/man8", ["man-pages/drbdmanage.8.gz"]),
                 ("/usr/share/dbus-1/system-services",
                     ["conf/org.drbd.drbdmanaged.service"]),
-                ("/var/drbd.d", []),
+                ("/var/lib/drbd.d", []),
                 ("/var/lib/drbdmanage", [])
                ],
     cmdclass={
